packages/ui/src/views/secrets/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from './new-secret/new-secret-form'
+export * from './create-secret/create-secret-page'
```

Comment:
i don't think this is a "page"... let's just call it `create-secret` since it's already scoped under `views` folder.